chart/tests/helm_template_generator.py

```diff
@@ -61,7 +88,7 @@ def validate_k8s_object(instance):
     validate.validate(instance)
 
 
-def render_chart(name="RELEASE-NAME", values=None, show_only=None, validate_schema=True):
```

Comment:
`validate_schema` was added recently to allow tests for CRDs but with this PR we have a way of pulling in the definition.
it does require a network call here fwiw: https://github.com/apache/airflow/pull/13209/files#diff-91361141bde70cbfc90142ebe2377ce918025d3343d9216e00e50c66f083b12cR65